src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,15 @@
 import React from 'react';
 import './MoviesList.scss';
+import moviesFromServer from '../../api/movies.json';
+import { MovieCard } from '../MovieCard';
 
 export const MoviesList = () => (
-  <>Put the list here</>
+  <>
+    <div className="movies">
+      {moviesFromServer.map(
+        movie => <MovieCard {...movie} key={movie.imdbId} />,
+      )
+      }
+    </div>
+  </>
```

Comment:
no need for React.Fragment here, you have only one element (div)
read about the correct usage [here](https://reactjs.org/docs/fragments.html)